analysis_driver/util/bash_commands.py

```diff
@@ -38,30 +38,26 @@ def seqtk_fqchk(fastq_file):
     return cmd
 
 
-def sickle_paired_end_in_place(fastq_file_pair):
+def fastq_filterer_an_pigz_in_place(fastq_file_pair, pigz_thread=10):
     """
-    Run sickle in paired end mode to do a very minimal trimming and filter reads shorter than 36 bases, i.e.
-    remove the adapter dimers flagged by bcl2fastq.
-    :param fastq_file_pair: A pair of fastqs to run sickle on
+    Run fastq filterer on a pair of fastq file which will remove any pair if one of the mate is shorter than 36 bases
     """
     if len(fastq_file_pair) != 2:
-        raise AnalysisDriverError('sickle_paired_end only supports paired fastq files')
+        raise AnalysisDriverError('fastq-filterer only supports paired fastq files')
 
     f1, f2 = sorted(fastq_file_pair)
     name, ext = os.path.splitext(f1)
-    of1 = name + '_sickle' + ext
-    ofs = name + '_sickle_single' + ext
-    lf = name + '_sickle.log'
+    of1 = name + '_fastq_filterer' + ext
     name, ext = os.path.splitext(f2)
-    of2 = name + '_sickle' + ext
+    of2 = name + '_fastq_filterer' + ext
+
     cmds = []
-    cmd = cfg['tools']['sickle'] + ' pe -f %s -r %s -o %s -p %s -s %s -q 5  -l 36  -x  -g -t sanger > %s'
-    cmds.append(cmd % (f1, f2, of1, of2, ofs, lf))
-    # replace the original files with the new files and remove the the single file to keep things clean
+    cmd = "set -o pipefail; %s --i1 %s --i2 %s --o1 >($pigz -p %s > %s) --o2 >($pigz -p %s > %s) --threshold 36"
```

Comment:
`$pigz` must be from an example Bash script! This should be `pigz -c`. Would also be good to make this (and the threshold) configurable.